lib/mbta.js

```diff
@@ -1,10 +1,16 @@
 'use strict';
 
-// Code here.
+let subwayLine = {};
+subwayLine['green'] = ['Government Center', 'Park Street', 'Boylston', 'Arlington', 'Copley', 'Hynes', 'Kenmore'];
+subwayLine['red'] = ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard', 'Porter', 'Davis', 'Alewife'];
+subwayLine['orange'] = ['North Station', 'Haymarket', 'Park Street', 'State', 'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills'];
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
 
-};
+
+const stopsBetweenStations = function (startLine, startStation, endLine, endStation) {
+
+}
+
```

Comment:
Then in here if you wanted to get the distance between two stations on the same line you could do something like 
```js
subwayLine[startLine].indexOf(startStation) - subwayLine[startLine].indexOf(endStation);
```